chat-components/src/components/prechatsurveypane/PreChatSurveyPane.tsx

```diff
@@ -104,7 +104,10 @@ function PreChatSurveyPane(props: IPreChatSurveyPaneProps) {
                 padding-top: 7px;
                 padding-bottom: 7px;
             }
-            .ac-pushButton { 
+            .ac-input.ac-toggleInput {
+                align-items: ${props.styleProps?.customToggleInputStyleProps?.alignItems ?? defaultPreChatSurveyPaneStyles.customToggleInputStyleProps?.alignItems} !important;
```

Comment:
Please add `customToggleInputStyleProps` to the corresponding place in the dev guide doc

Reply:
+1

Reply:
Please share the similar below information in the dev guide doc:
Using baseline would put the checkbox on top, using flex-end would put the checkbox at the bottom, and center is the default value.